### DISCUSSION TITLE
Incremental Mapping

### DISCUSSION THREAD

**Discussion by saimouli** [MrNeRF/LichtFeld-Studio#41] [Q&A] [answered]
Thanks for the great work. 

With the current code base would it be possible to incrementally insert images and do training/mapping? 

If not what code changes would you suggest to look into?

**Comment by MrNeRF** [comment, marked as answer]
No, currently this is not supported. 
Take a look at: https://github.com/MrNeRF/gaussian-splatting-cuda/blob/master/src/scene.cu
You just need to implement method, to load more data incrementally or you try lazy loading of images.

What what be your use case? What would this feature enable?

**Reply by saimouli** [reply]
Useful for mapping such as this work: https://www.youtube.com/watch?v=uzb-tVcPETE 

I am planning to use this codebase for my project and will be doing several improvements

**Comment by MrNeRF** [comment]
I see, very cool!
Are you on twitter? Then please send a DM to https://twitter.com/janusch_patas
Otherwise take a look at the git logs. There you gonna find my email contact.

**Reply by MrNeRF** [reply]
Btw, are you planning to contribute your changes or is it just for your personal project?
